eu_dgc_v1_schema.yaml

```diff
@@ -6,12 +6,30 @@ title: Digital Green Certificate
 description: Proof of vaccination, test results or recovery according to EU eHN, version 1.0, including certificate metadata;
   According to
   1) REGULATION OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL on a framework for the issuance, verification and acceptance of interoperable certificates on vaccination, testing and recovery to facilitate free movement during the COVID-19 pandemic (Digital Green Certificate) - https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0130
-  2) Document "Value Sets for the digital green certificate as stated in the Annex ...", abbr. "VS-2021-04-08" - https://webgate.ec.europa.eu/fpfis/wikis/x/05PuKg
-
+  2) Document "Value Sets for the digital green certificate as stated in the Annex ...", abbr. "VS-2021-04-14" - https://webgate.ec.europa.eu/fpfis/wikis/x/05PuKg
+  3) Guidelines on verifiable vaccination certificates - basic interoperability elements - Release 2 - 2021-03-12, abbr. "guidelines"
 type: object
 required:
+  - v
+  - dgcid
+  - is
   - sub
 properties:
+  v:
+    title: Schema version
+    description: Version of the schema, according to Semantic versioning (ISO, https://semver.org/ version 2.0.0 or newer) (viz. guidelines)
+    type: string
+    example: "1.0.0"
+  dgcid:
```

Comment:
What is the DGCID used for?

Reply:
Unique identifier of the (EVERY!) DGC (initially called UVCI (V for vaccination), later renamed to DGCI - by T-System, if I remember correctly).
Details, format and description see "Guidelines on verifiable vaccination certificates - basic interoperability elements - Release 2 - 2021-03-12"

Reply:
Ok, so payload identifier - what is it used for? I would recommend we don't require it. Some issuers may want to use the CWT signature (64 bytes) as the identifier, as this is unique anyway.